### PR TITLE
Model Input and Ouput Documentation

### PR DESCRIPTION
**Pull request by mansisandag**
## Proposed changes

Cleaning up documentation for model inputs and outputs. This is partial updates.

## Impact

Documentation updates.

## Types of changes

What types of changes does your code introduce to ABM?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] Built the documentation locally and confirmed the updates.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: docs/inputs.md [hunk 1]
```diff
@@ -25,54 +25,26 @@ The table below contains brief descriptions of the input files required to execu
...
-| visitor_outboundStopDuration.csv | Visitor model time-of-day offsets for outbound stops | CSV | Transportation Modelers | 
-| visitor_inboundStopDuration.csv | Visitor model time-of-day offsets for inbound stops | CSV | Transportation Modelers | 
-| **Airport Model (Derived from airport survey)** |  |  |  | 
-| [airport_purpose.csv](#airport_trip_purpose) | Airport model tour purpose frequency table | CSV | Transportation Modelers | 
-| [airport_party.csv](#airport_party_purpose) | Airport model party type frequency table | CSV | Transportation Modelers | 
-| [airport_nights.csv](#airport_nights) | Airport model trip duration frequency table | CSV | Transportation Modelers | 
-| [airport_income.csv](#airport_income) | Airport model trip income distribution table | CSV | Transportation Modelers | 
-| [airport_departure.csv](#airport_departure) | Airport model time-of-day distribution for departing trips | CSV | Transportation Modelers | 
-| [airport_arrival.csv](#airport_arrival) | Airport model time-of-day distribution for arriving trips | CSV | Transportation Modelers | 
+| mgra15.sbn | | SBN | |
```
Comment (JiaXu1024) [inline docs/inputs.md:46]: This should be shapefile type "mgra15.shp" @mansisandag

File: docs/inputs.md [hunk 2]
```diff
@@ -84,33 +56,35 @@ The table below contains brief descriptions of the input files required to execu
...
-| inputs_sandag_htm_<Scenario_Year>.xlsx | Contains all the required inputs ( in different sheets) for the Heavy Truck Model | Excel Workbook | |
-| FAF5_BaseAndFutureYears_Oct27_2023.csv | FAF5 Data (filtered) containing FAF flows for required years | CSV | |
+| HTM\inputs_sandag_htm_<Scenario_Year>.xlsx | Contains all the required inputs ( in different sheets) for the Heavy Truck Model | Excel Workbook | |
+| HTM\FAF5_BaseAndFutureYears_Oct27_2023.csv | FAF5 Data (filtered) containing FAF flows for required years | CSV | |
 | **Other** |  |  |  | 
 | [parametersByYears.csv](#parametersbyyearscsv) | Parameters by scenario years. Includes AOC, aiport enplanements, cross-border tours, cross-border sentri share. | CSV | Transportation Modelers | 
 | [filesByYears.csv](#filesbyyearscsv) | File names by scenario years. | CSV | Transportation Modelers | 
 | trip_XX.omx | Warm start trip table; XX is the TOD (EA, AM, MD, PM, and EV) | OMX | Transportation Modelers |
-| zone.term | TAZ terminal times | Space Delimited Text File | Transportation Modelers | 
+| zone.txt | TAZ terminal times | Space Delimited Text File | Transportation Modelers |
```
Comment (JiaXu1024) [inline docs/inputs.md:79]: the file name is 'zone_term.csv", please edit @mansisandag

File: docs/inputs.md [hunk 3]
```diff
@@ -2565,32 +2280,21 @@ oth; where Size_Class ranges 1-7.
 
 <a id="mobility_mgra"></a>
 
-### MGRAs at Mobility Hubs
-#### `MOBILITYHUBMGRA.CSV`
+### Zone Terminal Time
+#### `ZONE.txt`
```
Comment (JiaXu1024) [inline docs/inputs.md:2284]: is it zone_term.csv file?

File: docs/inputs.md [hunk 4]
```diff
@@ -1319,7 +1031,6 @@ regional_pass: 10.90
     
 </table>
 
-#### CVM Establishment Synthesis File ( MGRAEmpByEstSize.csv )
 ##### `MGRAEmpByEstSize.csv`
```
Comment (JiaXu1024) [inline docs/inputs.md:1034]: is this file under subheader Commercial vehicle model? please double check @mansisandag

**Review by JiaXu1024 — COMMENTED** [review]
@mansisandag Please address my comments

File: docs/inputs.md (hunk 1 above)
Comment (mansisandag) [reply docs/inputs.md:46]: Creating another PR because of branch issues to address this.

File: docs/inputs.md (hunk 2 above)
Comment (mansisandag) [reply docs/inputs.md:79]: Creating another PR because of branch issues to address this.

File: docs/inputs.md (hunk 3 above)
Comment (mansisandag) [reply docs/inputs.md:2284]: Creating another PR because of branch issues to address this.

File: docs/inputs.md (hunk 4 above)
Comment (mansisandag) [reply docs/inputs.md:1034]: Creating another PR because of branch issues to address this.

**Pull request closed without merging**